cmd/gb/gb_test.go

```diff
@@ -589,6 +589,17 @@ func TestInfoCmd(t *testing.T) {
 	gb.grepStdout(`^GB_GOROOT="`+regexp.QuoteMeta(runtime.GOROOT())+`"$`, "missing GB_GOROOT")
 }
 
+func TestInfoWithArgs(t *testing.T) {
+	gb := T{T: t}
+	defer gb.cleanup()
+
+	gb.tempDir("src")
+	gb.cd(gb.tempdir)
+	gb.run("info", "GB_PROJECT_DIR", "GB_GOROOT")
```

Comment:
could you add a test for missing something like 

`gb info GB_PROJECT_DIR GB_MISSING"

Should return one line of output and _not_ complain about GB_MISSING.